src/components/molecules/modals/SettingsModal/settings.scss

```
@@ -0,0 +1,60 @@
+#settings {
+  .nav-item {
+    padding: 0;
+  }
+
+  .pills-primary .nav-link {
+    background-color: darken(#fff, 2%) !important;
+    border-color: #77bd43;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    color: #212529;
+    text-align: left;
+
+    i {
+      padding-left: 0.5rem;
+    }
+
+    box-shadow: none;
```

Please move css property up.
```suggestion
    box-shadow: none;
    
    i {
      padding-left: 0.5rem;
    }
```